### PR TITLE
Milestone #5

### PR DESCRIPTION
**Pull request by edxco**
- Change the App component from a functional component to a Class component.
- Add the functionality to the buttons and the display

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status; Require changes 🚥 ♻️ 

Hi @edxco, congratulations on this milestone 😄 The PR and README look professional 👍 The code is clean. However, let us implement the following requirements as below:

- Kindly implement the handleClick functionality in the ButtonPanel component

[Optional]
- Kindly use the ES6 arrow syntax instead of the function keyword.

Once these are addresses, kindly submit another code review request. 
_Happy coding_ 👨‍💻 
[Daniel Larbi Addo](https://www.linkedin.com/mwlite/in/daniel-larbi-addo)

**Comment by addod19** [comment]
## Status: Invalid 

You have already submitted this same PR as MathMagicians Calculator Project. Where I have reviewed it accordingly but unfortunately you have also submitted it as Calculator Milestone5 Project. I will have to mark it as invalid so you can resolve the inconsistencies. Let us submit the right project for review to avoid this misunderstanding

**Comment by addod19** [comment]
Hi @edxco, you have submitted this PR for two projects namely: React Calculator Milestone5 and the Math Magicians Website This is not helping!!! Kindly check it out and resolve it

**Review by Forison — APPROVED** [review]
## Status::Approved✔️

# 👋 @edxco  

### You have done a good job moving the `handleclick` to the App.js as requested in the previous review🤗.

**[Optional but important]**

* [ ] There should be no logic for the setting the display value in the `App.js`, per this project requirements. All we need is to pass the state values ie `total`, `next` or `result` to `Display`. - 

- Check my comment below👇

#### Happy coding🤗

Addo Forison

File: src/components/App.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ class App extends Component {
 
   handleClick = (simbolOperation) => {
     const result = Calculate(this.state, simbolOperation);
-    console.log(result);
     this.setState(result);
   }
```
Comment (Forison) [inline src/components/App.js:20]: `<Display value={this.state.next || this.state.total || '0'} />`

no need for line 23-30

Note, we want to keep our components as dry as possible

**Review by Forison — CHANGES_REQUESTED** [review]

**Pull request merged**